inputfiles/overridingTypes.jsonc

```diff
@@ -1801,6 +1801,51 @@
                             "type": "MessageEvent"
                         }
                     ]
+                },
+                "constants": {
+                    "constant": {
+                        "CONNECTING": {
+                            "name": "CONNECTING",
+                            "readonly": true,
+                            "overrideType": "0"
+                        },
+                        "OPEN": {
+                            "name": "OPEN",
+                            "readonly": true,
+                            "overrideType": "1"
+                        },
+                        "CLOSING": {
+                            "name": "CLOSING",
+                            "readonly": true,
+                            "overrideType": "2"
+                        },
+                        "CLOSED": {
+                            "name": "CLOSED",
+                            "readonly": true,
+                            "overrideType": "3"
+                        }
+                    }
+                }
+            },
+            "EventSource": {
+                "constants": {
+                    "constant": {
+                        "CONNECTING": {
+                            "name": "CONNECTING",
```

Comment:
Thank you! One thing, you don't need to repeat this name and readonly field, the overrideType should be sufficient.

Reply:
thanks, I've removed these fields